crates/turborepo-repository/src/package_graph/builder.rs

```diff
@@ -651,4 +670,29 @@ mod test {
         }));
         assert_matches!(builder.build().await, Err(Error::DuplicateWorkspace { .. }));
     }
+
+    #[test]
+    fn test_missing_name_field_warning_message() {
```

Comment:
Okay to skip running this on Windows to avoid needing to deal with Windows paths. Only okay since we're testing string formatting
```suggestion
    #[cfg(unix)]
    fn test_missing_name_field_warning_message() {
```